session/sessionKey.go

```diff
@@ -0,0 +1,24 @@
+// This package is for all 'session' related code.
+package session
+
+import "github.com/pokt-network/pocket-core/crypto"
+
+/*
+"SessionKeyAlgo" function determines the sessionKey.
+ */
+func SessionKeyAlgo(devID string) []byte {
```

Comment:
Can we rename this function? I feel like the name is prone to confusion. something like GenerateSessionKey should be better.